api/types.go

```diff
@@ -293,6 +293,36 @@ type EmbeddingResponse struct {
 	Embedding []float64 `json:"embedding"`
 }
 
+// TokenizeRequest is the request sent by [Client.Tokenize].
```

Comment:
Wanted to follow a simple pattern similar to [tiktoken](https://github.com/openai/tiktoken)